templates/zerver/integrations/discourse.md

```diff
@@ -1,16 +1,35 @@
-1.  {!create-an-incoming-webhook.md!}
+# Zulip Discourse integration
 
-    The API keys for "Incoming webhook" bots are limited to only
-    sending messages via webhooks. Thus, this bot type lessens
-    the security risks associated with exposing the bot's API
-    key to third-party services.
+Get notifications for new Discourse posts in Zulip!
```

Comment:
I don't think this clearly states that the discourse post content is in the message in Zulip. It sounds like it might be something like "Lauryn posted in X" with a link to that discourse topic. Maybe something like the start of the Slack integration? "Forward new Discourse posts to Zulip!"